### DISCUSSION TITLE
Lost access to 25% of Roms

### DISCUSSION THREAD

**Discussion by rromney1** [eduardofilo/RG350_adam_image#273] [General]
### What happened?

I installed the new 2.0 image on my RG350. I love the new image but I lost access to 25% of my Roms. The image see my PCE CD Roms but crash when I try to play it. I lose access to my Sega CD Roms also they don’t show at all. Should I format my SD rom files and see if that helps or wait a patch for this issue?

### Device

RG350

### Version

v2.0

**Comment by eduardofilo** [comment]
Hi. From what you say, these are two different problems. The PCE CD problem may be related to the BIOS of this system. Did you install that file as discussed here?

https://github.com/eduardofilo/RG350_adam_image/wiki/En:-3.-Content-installation#bios

The few games I have from that system are working properly, what kind of crash is occurring, can you give more information?


The problem of SEGA CD games not appearing in the listings will probably be easier to solve. But you have to provide more information to get to the cause of the problem. Again, I don't have any problems viewing or booting the games I have from that system. Please post screenshots of what you see or don't see and the contents of the directories where the ROMs are (made from Commander for example, or from your computer's file explorer if you mount the external card in the PC).

**Comment by rromney1** [comment]
![image](https://user-images.githubusercontent.com/39176835/176556793-1a00fb73-ef90-464f-883e-2f818667a239.jpeg)

**Comment by rromney1** [comment]
These are the Roms for sega cd but when the image don't pick them up

**Comment by rromney1** [comment]
![image](https://user-images.githubusercontent.com/39176835/176557177-07804ecc-ec7f-41aa-8395-43f8fc9d9dd4.jpeg)
Download the syscard3.pce still not working. I might downgrade back to 1.4.2 it read all my Roms no issues.

**Comment by eduardofilo** [comment]
**Problem with SEGACD**

You seem to have enabled the option to hide extensions in Windows file explorer. Please disable that option because it could hide the problem. Or send the screenshot from the console Commander application.
The extensions supported in SEGACD have not changed between the 1.4.2 and 2.0 versions of the image, nor has the SimpleMenu version.
I see all my ROMs without problems, although it is true that they are all in chd, not bin.
Is your external card in FAT32 or exFAT format?

**Problem with PCE-CD**

Those two games work perfectly for me on Adam 2.0.
I just checked that the md5 hash of the syscard3.pce file has changed in the core_info file. Before it was `390815d3d1a184a9e73adc91ba55f2bb` and now it is `38179df8f4ac870017db21ebcbf53114`. Although it must not be very important, because mine has as hash `ff1a674273fe3540ccef576376407d1d` and the games work for me.
